library/src/blas3/rocblas_trmm_strided_batched.cpp

```diff
@@ -0,0 +1,373 @@
+
+                if(layer_mode & rocblas_layer_mode_log_bench)
+                {
+                    log_bench(handle,
+                              "./rocblas-bench -f trmm_strided_batched -r",
+                              rocblas_precision_string<T>,
+                              "--side",
+                              side_letter,
+                              "--uplo",
+                              uplo_letter,
+                              "--transposeA",
+                              transa_letter,
+                              "--diag",
+                              diag_letter,
+                              "-m",
+                              m,
+                              "-n",
+                              n,
+                              LOG_BENCH_SCALAR_VALUE(alpha),
+                              "--lda",
+                              lda,
+                              "--stride_a",
+                              stride_a,
+                              "--ldb",
+                              ldc,
+                              "--stride_b",
+                              stride_c,
+                              "--batch_count",
+                              batch_count);
+                }
+            }
+            else
+            {
+                if(layer_mode & rocblas_layer_mode_log_trace)
+                    log_trace(handle,
+                              rocblas_trmm_strided_batched_name<T>,
+                              side,
+                              uplo,
+                              transa,
+                              diag,
+                              m,
+                              n,
+                              log_trace_scalar_value(alpha),
+                              a,
+                              lda,
+                              stride_a,
+                              c,
+                              ldc,
+                              stride_c,
+                              batch_count);
+            }
+
+            if(layer_mode & rocblas_layer_mode_log_profile)
+            {
+                log_profile(handle,
+                            rocblas_trmm_strided_batched_name<T>,
+                            "side",
+                            side_letter,
+                            "uplo",
+                            uplo_letter,
+                            "transa",
+                            transa_letter,
+                            "diag",
+                            diag_letter,
+                            "m",
+                            m,
+                            "n",
+                            n,
+                            "lda",
+                            lda,
+                            "stride_a",
+                            stride_a,
+                            "ldb",
+                            ldc,
+                            "stride_b",
+                            stride_c,
```

Comment:
Not clear why a stride_c search and replace with stride_b wasn't done?     C API uses c but doxygen b.  I guess for consistency with original code this was done.

Reply:
change to consistently use b, ldb, stride_b in place of sometimes c, ldc, stride_c